src/NzbDrone.Core/Datastore/ConnectionStringFactory.cs

```diff
@@ -60,9 +76,9 @@ private static DatabaseConnectionInfo GetConnectionString(string dbPath)
             return new DatabaseConnectionInfo(DatabaseType.SQLite, connectionBuilder.ConnectionString);
         }
 
-        private DatabaseConnectionInfo GetPostgresConnectionString(string dbName)
+        private DatabaseConnectionInfo GetPostgresConnectionInfoFromIndividualValues(string dbName)
```

Comment:
I think `BuildPostgresConnectionString` would be clear enough without being too verbose.